test/fake_3par_data.py

```diff
@@ -134,28 +139,119 @@
     'qos_name': None,
     'compression': None,
     'snapshots': [],
-    'node_mount_info': {OTHER_NODE_ID: ['Fake_Mount_id']},
+    'node_mount_info': {OTHER_NODE_ID: ['Fake-Mount-ID']},
     'path_info': path_info,
-    'mount_conflict_delay': 3}
-
-snapshot1 = {'name': SNAPSHOT_NAME1,
-             'id': SNAPSHOT_ID1,
-             'parent_id': VOLUME_ID,
-             'expiration_hours': '10',
-             'retention_hours': '10'}
-
-snapshot2 = {'name': SNAPSHOT_NAME2,
-             'id': SNAPSHOT_ID2,
-             'parent_id': VOLUME_ID,
-             'expiration_hours': '5',
-             'retention_hours': '5'}
-
-snapshot3 = {'name': SNAPSHOT_NAME3,
-             'id': SNAPSHOT_ID3,
-             # This is a child of snapshot1
-             'parent_id': SNAPSHOT_ID1,
-             'expiration_hours': '5',
-             'retention_hours': '5'}
+    'mount_conflict_delay': 3,
+    'is_snap': False
+}
+
+volume_mounted_twice_on_this_node = {
+    'name': VOLUME_NAME,
+    'id': VOLUME_ID,
+    'display_name': VOL_DISP_NAME,
+    'size': 2,
+    'host': FAKE_DOCKER_HOST,
+    'provisioning': THIN,
+    'flash_cache': None,
+    'qos_name': None,
+    'compression': None,
+    'snapshots': [],
+    'node_mount_info': {THIS_NODE_ID: ['Fake-Mount-ID', 'Fake-Mount-ID']},
+    'path_info': path_info,
+    'mount_conflict_delay': 3,
+    'is_snap': False
+}
+
+snap1_metadata = {
+    'name': SNAPSHOT_NAME1,
+    'id': SNAPSHOT_ID1,
+    'parent_name': VOLUME_NAME,
+    'parent_id': VOLUME_ID,
+    'expiration_hours': '10',
+    'retention_hours': '10'
+}
+
+snap1 = {
+    'name': SNAPSHOT_NAME1,
+    'id': SNAPSHOT_ID1,
+    'display_name': SNAPSHOT_NAME1,
+    'parent_id': VOLUME_ID,
+    'ParentName': VOLUME_NAME,
+    'expiration_hours': '10',
+    'retention_hours': '10',
+    'is_snap': True,
+    'size': 2,
+    'snap_metadata': snap1_metadata,
+    'snapshots': []
+}
+
+snap2_metadata = {
+    'name': SNAPSHOT_NAME2,
+    'id': SNAPSHOT_ID2,
+    'parent_name': VOLUME_NAME,
+    'parent_id': VOLUME_ID,
+    'expiration_hours': '10',
+    'retention_hours': '10'
+}
+
+snap2 = {
+    'name': SNAPSHOT_NAME2,
+    'id': SNAPSHOT_ID2,
+    'display_name': SNAPSHOT_NAME2,
+    'parent_id': VOLUME_ID,
+    'ParentName': VOLUME_NAME,
+    'expiration_hours': '5',
```

Comment:
expiration_hours and retention_hours are part of snap_metadata and not of snapshot dictionary

Reply:
ok